cloud/gcp/integrations-gcp.tf

```diff
@@ -19,15 +19,12 @@ resource "signalfx_org_token" "gcp_integration" {
   }
 }
 
-data "signalfx_gcp_services" "gcp_services" {
-}
-
 resource "signalfx_gcp_integration" "gcp_integration" {
   name        = local.integration_name
   enabled     = var.enabled
   named_token = signalfx_org_token.gcp_integration.name
   poll_rate   = var.poll_rate
-  services    = coalescelist(var.services, data.signalfx_gcp_services.gcp_services.services[*].name)
+  services    = var.services
```

Comment:
Maybe also update the constraint in `versions.tf` to `>= 8.0.0`, otherwise it will be a breaking change for stacks using the old version: no services instead of all services.

https://github.com/claranet/terraform-signalfx-integrations/blob/b9d3a6870e33c0c9378d29e9fa495dd6aebc9361/cloud/gcp/versions.tf#L5

Reply:
done

Reply:
README needs to be updated too, there's a script for this: https://github.com/claranet/terraform-signalfx-integrations/blob/a1abf9dfbad84dad97d62c507ea4a75538fb8b61/scripts/docs.sh

We should add checks for this in Github Actions...

Reply:
done